NOTES-WINDOWS.md

```diff
@@ -39,7 +39,7 @@ these are required as well:
 ### Perl
 
 We recommend Strawberry Perl, available from <http://strawberryperl.com/>
-Please read NOTES.PERL for more information, including the use of CPAN.
+Please read [NOTES-PERL.md](NOTES-PERL.md) for more information, including the use of CPAN.
```

Comment:
Could you please use the reference style link syntax? It's preferred for readability. (Also use the implicit link name `[]`)